### PR TITLE
Mathjax output SVG instead of css

### LINKED ISSUE #7848: Strange MathJax rendering when using tablet (Samsung Galaxy Tab S6)
###### Reproduction Steps

1. Reviewing cards with MathJax.

###### Expected Result
MathJax being rendered regularly, like it is in my phone Samsung Note 9.

###### Actual Result
When reviewing in the tablet (Samsung Galaxy Tab S6), MathJax is rendered strangely:
SmartSelect_20201208-054928_AnkiDroid
SmartSelect_20201208-061155_AnkiDroid
SmartSelect_20201208-083607_AnkiDroid

###### Debug info
AnkiDroid Version = 2.15alpha11
Android Version = 10
ACRA UUID = 38f97252-6295-4be3-b0f4-864eaa211b02
Scheduler = std2
Crash Reports Enabled = true

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This should hopefully fixes #7848, i.e. on samsung tablets, formulas are not correctly displayed.
The downsides are mentionned in http://docs.mathjax.org/en/latest/output/svg.html :
* it copy as image and not as text
* change in numbering of tags of equation (normally, cards are small enough that's not a problem)

I tested and it works perfectly on my tablet.
Of course, I also hope that it'll work for @hugomarins , but I believe we'll need to merge so that he can test easily.

@hgiesel, I'd love if you accept to review this. I can't ask you using github interface because you're not part of the organization, but it seems you are the one knowing best this code

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
I'm willing to take this step - alpha releases were made for this kind of "should work? might work? let's try it" default stance changes followed by a call for testers.

So I will approve this. I'd like to mark it for a quick check from @hgiesel even though it's more of a "be aware we're trying it..." type of check so that if he sees anything in the next alpha he'll have his eyes peeled etc

@hgiesel if this seems okay to you - even if a bit "bold" - please approve and I'll merge it and generate an alpha to see how it goes over.

If it fails, we can try an advanced preferences toggle I suppose

**Comment by mikehardy** [comment]
@hgiesel I just invited you to be a reviewer in the org, I can't actually request your review without it. Please understand no obligation or expectation comes with acceptance or rejection of that, it's just GitHub workflow powers in my opinion and nothing more, though I do always lean towards empowering people in case their interested :-). Cheers

**Comment by hgiesel** [comment]
It looks good to me! There's just two things I'd like to mention:

1. I don't think you can zoom into cards in AnkiDroid, can you? I'd would have liked to see how the SVGs behave in that case.
2. [On the MathJax 2 documentation for output modes](https://docs.mathjax.org/en/v2.7-latest/output.html), it is mentioned that the SVG output does not rely on the Math fonts. So it might be possible to remove them, if we would really go all in for SVG output.

**Comment by mikehardy** [comment]
Interesting - good thoughts -

1- pinch-zoom is definitely a thing in one of my decks that has cards with images on it. Not sure about double-tap to zoom or anything. @Arthur-Milchior did this ever work with Mathjax, and/or is it breaking now? (a comparison of 2.13.5 / 2.14.2 / this PR would be interesting)

2- if we're doing a hard switch then the entire assets/mathjax/chtml directory goes away I think? And that's nearly 400 kilobytes of probably-uncompressable fonts, so that's cool. Pending the answer to 1 above (and our decision on it - make this an advanced pref toggle or just hard switch to svg) if we do a hard switch I would want to see that removed in this PR yes

**Comment by Arthur-Milchior** [comment]
Using two fingers to zoom works as well as it always worked. I have never seen double click to work for zooming. The fact that gesture may have other meaning may potentially block it, I don't know

**Comment by Arthur-Milchior** [comment]
I removed fonts. Are there other things to remove ?

I tested and math still works on my tablet

**Comment by mikehardy** [comment]
Great news re: pinch-zoom

I could be wrong but I think the entire chtml directory could possibly be purged? I am no mathjax expert by any stretch though. Maybe you know @hgiesel  ?

This seems like it might work with no regressions (famous last words)

**Comment by hgiesel** [comment]
When updating to MathJax 3 for Anki, I used [this guide](https://openbase.io/js/mathjax/documentation#reducing-the-size-of-the-components-directory), which also mentions the font files for SVG output. Now it seems like the output directory (which includes the chtml directory) so I guess Git should actually remove it automatically.

**Review by hgiesel — APPROVED** [review]

**Comment by mikehardy** [comment]
Great - let's do this, thanks @Arthur-Milchior for the work and @hgiesel for the help

I will keep my eyes/ears open in case there reports of incompatibility, but @hugomarins we should have beautiful mathjax everywhere with the next alpha

**Comment by hugomarins** [comment]
Thanks @mikehardy @Arthur-Milchior and @hgiesel . I'll keep testing and letting you know.
